src/app-layout/__tests__/app-layout-navigation.test.tsx

```diff
@@ -1,9 +1,9 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-/* eslint simple-import-sort/imports: 0 */
```

Comment:
Primary reason for this change. We do not have side-effects in import order anymore, so we can remove this eslint rule exception